docs/development/remote-development.md

```diff
@@ -38,6 +38,11 @@ You can use [Gitpod](https://gitpod.io/) for light development work like:
 - Editing the docs
```

Comment:
Lines 36-39 can be edited we know Gitpod and GitHub Codespaces behave the same for development work.